runtime/src/iree/hal/cts/semaphore_submission_test.h

```diff
@@ -87,10 +87,10 @@ TEST_P(semaphore_submission_test, SubmitWithWait) {
       signal_payload_values,
   };
 
-  IREE_ASSERT_OK(
-      iree_hal_device_queue_execute(device_,
-                                    /*queue_affinity=*/0, wait_semaphores,
-                                    signal_semaphores, 1, &command_buffer));
+  IREE_ASSERT_OK(iree_hal_device_queue_execute(
+      device_,
+      /*queue_affinity=*/0, wait_semaphores, signal_semaphores, 1,
+      &command_buffer, NULL));
```

Comment:
Could add comments for `/*binding_tables=*/NULL` to the uses here in this file.

Could we also add tests using binding tables / indirect execution somewhere?